### PR TITLE
Fail fast

### PR DESCRIPTION
**Pull request by chuff**

### CODE REVIEW

File: modules/cmpapi/.mocharc.json [hunk 1]
```diff
@@ -32,9 +32,8 @@
     "./test/encoder/section/UsCtV1.test.ts",
     "./test/encoder/section/UsNatV1.test.ts",
     "./test/encoder/section/UsUtV1.test.ts",
-    "./test/encoder/section/UspV1.test.ts",
-    "./test/encoder/section/UspV1.test.ts",
     "./test/encoder/section/UsVaV1.test.ts",
+    "./test/encoder/section/UspV1.test.ts",
```
Comment (patmmccann) [inline modules/cmpapi/.mocharc.json:36]: will this create a merge conflict with #54?

File: modules/cmpapi/src/encoder/GppModel.ts [hunk 2]
```diff
@@ -226,94 +195,100 @@ export class GppModel {
...
+          } else if (sectionIds[i] === TcfEuV2.ID) {
+            let section = new TcfEuV2(encodedSection);
+            sections.set(TcfEuV2.NAME, section);
+          } else if (sectionIds[i] === UspV1.ID) {
+            let section = new UspV1(encodedSection);
+            sections.set(UspV1.NAME, section);
+          } else if (sectionIds[i] === UsNatV1.ID) {
+            let section = new UsNatV1(encodedSection);
+            sections.set(UsNatV1.NAME, section);
+          } else if (sectionIds[i] === UsCaV1.ID) {
```
Comment (patmmccann) [inline modules/cmpapi/src/encoder/GppModel.ts:251]: as of #54 all these v1 need to be removed

File: modules/cmpapi/.mocharc.json (hunk 1 above)
Comment (chuff) [reply modules/cmpapi/.mocharc.json:36]: Resolved

File: modules/cmpapi/src/encoder/GppModel.ts (hunk 2 above)
Comment (chuff) [reply modules/cmpapi/src/encoder/GppModel.ts:251]: I'll address this in a separate PR.

File: modules/cmpapi/src/encoder/field/UsNatV1Field.ts [hunk 3]
```diff
@@ -20,3 +20,24 @@ export enum UsNatV1Field {
   GPC_SEGMENT_INCLUDED = "GpcSegmentIncluded",
   GPC = "Gpc",
 }
+
+export const USNATV1_CORE_SEGMENT_FIELD_NAMES = [
```
[the remark this answers is not in the document]
Comment (chuff) [reply modules/cmpapi/src/encoder/field/UsNatV1Field.ts:24]: Yeah, I'll address this in a separate PR

**Comment by schavvaOt** [comment]
Looks good to me

**Review by schavvaOt — APPROVED** [review]

**Review by patmmccann — APPROVED** [review]

**Pull request merged**